src/sass/components/_loader.scss

```diff
@@ -5,60 +5,132 @@ Loader
 Loader and variations
 
 Markup:
-<a href="#" class="btn btn- simple_loader {{modifier_class}}">Button</a>
+<a href="#" class="btn loader {{modifier_class}}">Button</a>
 
 .loader--center          - Loader without button label
+.loader--large           - Loader large
+.loader--small           - Loader small
+.loader--xsmall          - Loader xsmall
 .loader--gray            - Loader gray
+.btn--large              - Large size
+.btn--small              - Small button with loader
+.btn--xsmall             - XSmall button with loader
+
 
 Styleguide 15.0
 
 */
 
-.simple_loader {
-  position:relative;
+@mixin loader-small {
+  padding-right: 30px;
+
+  &.loader--center {
+    padding-right: 15px;
+  }
+
+  &:after {
+    width: 8px;
+    height: 8px;
+  }
 }
 
-.simple_loader:after {
-  border: 2px solid transparent;
-  border-radius: 50%;
-  border-top: 2px solid white;
-  border-right: 2px solid white;
-
-  position: absolute;
-  top: 0px;
-  bottom: 0px;
-  right: 10px;
-  margin: auto;
-
-  width: 10px;
-  height: 10px;
-  content: "";
-  display: inline-block;
-  animation: spin 1s linear infinite;
-  margin-left: 20px;
+@mixin loader-xsmall {
+  &:after {
+    width: 8px;
+    height: 8px;
+    border-width: 1px;
+  }
 }
 
-.simple_loader.loader--center {
-  color: transparent;
+@mixin loader-large {
   &:after {
-    left: 0px;
-    right: 0px;
-    margin: auto;
+    width: 12px;
+    height: 12px;
+  }
+}
+
+.loader.btn,
+.loader[type=button],
+.loader[type=submit],
+button.loader {
+  pointer-events: none;
+  position:relative;
+  &:after {
+    border: 2px solid transparent;
+    border-radius: 50%;
+    border-top: 2px solid white;
+    border-right: 2px solid white;
+
     position: absolute;
     top: 0px;
     bottom: 0px;
+    right: 10px;
+    margin: auto;
+
+    width: 10px;
+    height: 10px;
+    content: "";
+    display: inline-block;
+    animation: spin 1s linear infinite;
```

Comment:
Talked to Hunter, let's do this for easing function: `cubic-bezier(0.35, 0.55, 0.85, 0.95)`